std_msgs/Header header
ObstacleDetection[] detections